doc/source/whatsnew/v0.16.0.txt

```diff
@@ -117,6 +117,21 @@ Enhancements
 - Added ``StringMethods.ljust()`` and ``rjust()`` which behave as the same as standard ``str`` (:issue:`9352`)
 - ``StringMethods.pad()`` and ``center()`` now accept `fillchar` option to specify filling character (:issue:`9352`)
 
+
+
```

Comment:
ok move this to the api section snd show s code block of the existing behavior (and thrn out this after)